Source/Timeline/TimelineView.swift

```diff
@@ -42,7 +42,7 @@ public class TimelineView: UIView, ReusableView {
 
   var style = TimelineStyle()
 
-  var verticalDiff: CGFloat = 45
+  var verticalDiff: CGFloat = 100
```

Comment:
Is this an intentional change for this pull request? It looks to me, that it is related to changing timeline scale and appeared here occasionally. If so, please remove it.

Reply:
Yep, it shouldn't be in this pull request! I should stop trying to look at two issues at once!